client/hotrod-client/src/test/java/org/infinispan/client/hotrod/near/EvictInvalidatedNearCacheTest.java

```diff
@@ -37,7 +37,7 @@ public void testEvictAfterReachingMax() {
       assertClient.get(3, "v1").expectNearGetNull(3).expectNearPutIfAbsent(3, "v1");
       assertClient.get(2, "v1").expectNearGetValue(2, "v1");
       assertClient.get(3, "v1").expectNearGetValue(3, "v1");
-      assertClient.get(1, "v1").expectNearGetNull(1).expectNearPutIfAbsent(1, "v1");
+//      assertClient.get(1, "v1").expectNearGetNull(1).expectNearPutIfAbsent(1, "v1");
```

Comment:
Unless there is a good reason to keep this commented (with a Jira to potentially uncomment it), I'd remove it altogether